### PR TITLE
Editing Pages - Caching page content

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**

### CODE REVIEW

File: content/docs/common-areas/caching.md [hunk 1]
```diff
@@ -1,65 +1,62 @@
...
 ### Cache Bins
 
-In Drupal, the cache storage is separated into bins. These bins are used to store cached data of different types. Some of the common cache bins are as follows:
+In Drupal, cache storage is organized into bins, each storing different data types. Common bins include **bootstrap**, **render**, **data**, and **discovery**.
 
-1. bootstrap: Data needed from the beginning to the end of most requests, that has a very strict limit on variations and is invalidated rarely.
-2. render: Contains cached HTML strings like cached pages and blocks, can grow to large size.
-3. data: Contains data that can vary by path or similar context.
-4. discovery: Contains cached discovery data for things such as plugins, views_data, or YAML discovered data such as library info.
+Custom cache bins can be created by defining them in the `modulename.services.yml` file.
```
Comment (hussainweb) [inline content/docs/common-areas/caching.md:40]: Removing the description of common bins and then mentioning this detail seems unnecessary. Let's just say that modules may create cache bins for their own use.

Comment (hussainweb) [reply content/docs/common-areas/caching.md:40]: Also, there is inconsistency between newlines. Here, the next line is rendered in the same paragraph but it is on a separate line in markdown. We don't use newlines for this in the rest of the handbook.

File: content/docs/common-areas/caching.md [hunk 2]
```diff
@@ -1,65 +1,62 @@
...
-
-Read more about cache bins from [here](https://api.drupal.org/api/drupal/core!core.api.php/group/cache/8.2.x)
+Read more about [cache bins](https://api.drupal.org/api/drupal/core%21core.api.php/group/cache/).
 
 ## Cache Systems
 
-By default, Drupal stores the cache data in the database. Again, this may be convenient but results in poor performance. This is why Drupal is usually configured to store the cached data in external stores such as Memcache and Redis.
-
-Configuring Memcache or Redis can be categorised under Database Caching. These can be used to cache database data, API calls or page rendering chunks in RAM to increase the application performance.
+Drupal's default cache storage is in the database, which is convenient but can hinder performance. To boost application performance, Drupal is typically configured to use external cache stores like **Memcache** or **Redis**. These tools can cache database data, API calls, or page rendering chunks in RAM, enhancing overall efficiency.
```
Comment (hussainweb) [inline content/docs/common-areas/caching.md:46]: > These tools can cache database data...

I know you have rephrased but the meaning has changed. You have swapped the subject and the object and this implies that memcache and redis actively do something to cache database, API, etc... This is not true, of course.

File: content/docs/common-areas/caching.md [hunk 3]
```diff
@@ -1,65 +1,62 @@
...
+[Memcache](https://memcached.org/) is a free, open-source, high-performance memory caching system. It can be integrated into a Drupal website using the [Memcache module](https://www.drupal.org/project/memcache), which also offers an administrative overview of Drupal's interaction with Memcached and provides performance stats.
 
 ### Redis
 
-Similar to memcache, [redis](https://redis.io/docs/about/) is another open source (BSD licensed), in-memory data structure store used as a database, cache, message broker, and streaming engine. Redis can be configured on a drupal website with the help of the [redis contributed module](https://www.drupal.org/project/redis). Both tools can be used to implement or add another layer of caching on a website to improve the performance but the selection between the two may depend upon the requirement and the features that they ship with.
+[Redis](https://redis.io/docs/about/) is a versatile, open-source (BSD licensed) in-memory data structure store used as a database, cache, message broker, and streaming engine. It can be configured on a Drupal website with the [Redis module](https://www.drupal.org/project/redis).
```
Comment (hussainweb) [inline content/docs/common-areas/caching.md:54]: Let's fix this. The license has changed.

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: content/docs/common-areas/caching.md (hunk 1 above)
Comment (Yemaneberhan-Lemma) [reply content/docs/common-areas/caching.md:40]: > Removing the description of common bins and then mentioning this detail seems unnecessary. Let's just say that modules may create cache bins for their own use.

@hussainweb: I shared the link below that line, and the description of each common bin is there.

Comment (hussainweb) [reply content/docs/common-areas/caching.md:40]: I don't see the description. See https://deploy-preview-205--axelerant-engg-handbook.netlify.app/docs/common-areas/caching/#cache-bins.

**Comment by Yemaneberhan-Lemma** [comment]
@hussainweb: Based on your comments, changes are made to the content.

**Review by hussainweb — APPROVED** [review]

**Pull request merged**